AGENTS.md

```
@@ -1,11 +1,304 @@
+
+---
+
+# Special Terminology
+
+## Cleave
+
+Remove **all words contained in parentheses**.
+
+Example:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)